tests/integration/records/unload-test.js

```diff
@@ -136,2827 +138,2873 @@ module('integration/unload - Unloading Records', {
       book: Book,
       spoon: Spoon,
       show: Show,
-    });
-  },
+    };
+    Object.keys(models).forEach(key => owner.register(`model:${key}`, models[key]));
```

Comment:
IMHO the `models` iteration here is a tad "cute", I think it would be easier to read as multiple lines like `owner.register(`model:foo`, Foo)` (and remove the `models` POJO just above)...